devices/flash-zynq7000/flashcfg.c

```diff
@@ -0,0 +1,104 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system loader
+ *
+ * Common Flash Interface for flash drive
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Hubert Buczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#include "flashcfg.h"
+#include <lib/errno.h>
+
+
+#define MANUFACTURER_ID_SPANSION 0x1
+#define SPANSION_MEM256          0x19
+#define SPANSION_MEM128          0x18
+
+
+static void flashcfg_spansionCmds(flash_info_t *info)
+{
+	info->cmds[flash_cmd_rdid].opCode = FLASH_CMD_RDID;
+	info->cmds[flash_cmd_rdid].size = 1;
+	info->cmds[flash_cmd_rdid].dummyCyc = 24;
```

Comment:
It would be best to just place it in .rodata as static const table, but I don't see way to do this and keep readability (i.e. tying enum and define in clear way), so I propose this middle-ground: 
```suggestion
	info->cmds[flash_cmd_rdid] = { FLASH_CMD_RDID, 1, 24 };
```